packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItemSuggestion.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { classNamesFunction, styled } from '../../../Utilities';
+import { ITagItemSuggestionProps, ITagItemSuggestionStyleProps, ITagItemSuggestionStyles } from './TagPicker.types';
+import { getStyles } from './TagItemSuggestion.styles';
+
+const getClassNames = classNamesFunction<ITagItemSuggestionStyleProps, ITagItemSuggestionStyles>();
+
+export const TagItemSuggestionBase = (props: ITagItemSuggestionProps) => {
```

Comment:
can we split this out to its own base file?

Reply:
Same as below.